### DISCUSSION TITLE
Construction for renewables

### DISCUSSION THREAD

**Discussion by dt-woods** [NETL-RIC/ElectricityLCI#229] [Q&A]
In preparation for the new renewable inventories where construction and O&M are separated, I came across an example of construction already in the baseline.

Not sure why, but it seems "Electricity - SOLAR - ISO-NE" links to NGCC power plant construction. I can't find any other examples of renewables linked to construction. Any thoughts where this came from?

https://www.lcacommons.gov/lca-collaboration/Federal_LCA_Commons/US_electricity_baseline/dataset/PROCESS/61e61a66-c44f-3199-adbd-2e8eb658cd7c

**Comment by dt-woods** [comment]
The documentation text for the "power plant construction - ngcc_const" is a little choppy. Could also use some help in ironing it out.

Here's the original:

> The cradle-to-gate inventory for the construction of the fossil fuel generators (coal, gas, and oil). The inventories represent the construction of either natural gas combined cycle or coal power plants using an economic input output life cycle inventory model (https://doi.org/10.1016/j.jclepro.2017.04.150) and the construction costs for these plants as detailed by NETL techno-economics analysis (https://www.netl.doe.gov/energy-analysis/details?id=729). These type different types of power plants are then assigned to the various power plants according to the plant they most resemble. The inventories are then scaled according to power plant capacity relative to the source power plants and the allocated over an assumed 30 year life. The single year of impacts are then divided by the generation of the year selected in the model configuration. This process was created with ElectricityLCI (https://github.com/USEPA/ElectricityLCI) version 1.0.1 using the ELCI_1 configuration.

Areas of confusion start with sentence three, "_These type different types of power plants are then assigned to the various power plants according to the plant they most resemble._" Does this include non-coal, non-gas, non-oil facilities, such as solar (see above). 

Next is "_The inventories are then scaled according to power plant capacity relative to the source power plants and the allocated over an assumed 30 year life._" How does this work with a functional unit of 1 item for this UP? Does this explain the fractional input used in downstream processes (e.g., 3.142e-6 items)? If so, that's not clear from this description; it sounds like the inventory of this process is scaled.

Also, "_The single year of impacts are then divided by the generation of the year selected in the model configuration._" I'm assuming "single year of impacts" means emission factors (and not life-cycle impacts). For the 2016 baseline, is this just divided by 2016 generation? It's rather evasively described here.

**Reply by m-jamieson** [reply]
The intent is not to assign these to renewable technologies. Mostly it's assigning gas power plant constrution to say DFO or RFO-fired plants because those are expected to be similar in size...certainly much moreso than coal. Biomass plants would more closely resemble coal plants. 

For the solar plant it would take some investigation, but there is petroleum fuel input, so maybe the prime mover code says gas turbine or something like that. I'm sure in reality, there's a DFO/RFO-fired turbine or engine also on-site with the solar that operates some of the time.

We have inventories for full 600+MW power plants - both coal and NGCC. We scale that inventory by that net power output so that we can scale those inventories according to the actual plant capacity. So a 300 MW coal plant would get about half of the inventory of the 600 MW plant.

And then that inventory is divided by 30 years (I think) to get a single year of that plant inventory...and then finally divided by the 2016 generation to get construction impacts on a MWh basis.

So X kg CO2/600 MW plant * 300 MW / 30 years * 1 yr/ X MWh (2016)

**Reply by dt-woods** [reply]
That's interesting. What I'm seeing is that the scaling for a "matched plant" follows something like this:

$$
\mathrm{emission}, kg/MWh = \frac{\mathrm{emission}, kg}{\mathrm{lifetime}, yr\times\mathrm{net\ gen}, MWh/yr}\times\frac{\mathrm{capacity\ actual}, MW}{\mathrm{capacity\ modeled}, MW}
$$

For the renewable inventories that we are updating, my thought was to scale construction-based emissions using this equation:

$$
\mathrm{emission}, kg/MWh = \frac{\mathrm{emission}, kg}{\mathrm{lifetime}, yr\times\mathrm{capacity}, MW \times 8760, hr/yr}
$$

When you did the 2020 construction and O&M inventories for solar PV, solar thermal, and wind, did you model the inventories after a capacity-specific plant and use the first equation, or did you scale construction to be incorporated into O&M (e.g., as a sub-process)? I'm thinking something like the following diagram:

![Renewable Mix](https://github.com/USEPA/ElectricityLCI/assets/29642997/dd9532a5-5d0a-4179-9294-6b36bad74b2e)

Also, is it acceptable to scale construction emissions purely on plant capacity? I'm thinking of cases where the footprint of a building is the same, but the size of the motor it houses is different. Both require the same amount of concrete for the footers and driveway and both need pipes for water intakes and outtakes.
